src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/shipped.html.twig

```diff
@@ -0,0 +1,4 @@
+<span class="ui green{% if attached is defined and attached == true %} top attached{% endif %} label">
+    <i class="plane icon"></i>
+    {{ value|trans }}
+</span>
```

Comment:
Instead of multiple templates we can go with a one template with the state -> color array, like [here](https://github.com/Sylius/Sylius/pull/10233/files#diff-44345c41ccef06b29bd3dfaacbca9cd7R2)